app/react/portainer/environments/wizard/EnvironmentsCreationView/WizardDocker/APITab/APIForm.tsx

```diff
@@ -70,7 +73,15 @@ export function APIForm({ onCreate, isDockerStandalone }: Props) {
             />
           </FormControl>
 
-          <TLSFieldset />
+          <TLSFieldset
+            values={values.tlsConfig}
+            onChange={(value) =>
+              Object.entries(value).forEach(([key, value]) =>
```

Comment:
```
tlsConfig => setValues(values => {
	...values
	tlsConfig,
})
```

Reply:
Do you mean `setFieldValue` instead of `setValues`? Actually this piece of code copy from `AuthFieldset` component

Reply:
I mean to use `setValues` instead of `setFieldValue`. but actually, since `tlsConfig` is a key, you can just do `setFieldValue('tlsConfig', value)`